README.md

```diff
@@ -50,6 +50,11 @@ Possible networks: `etc`, `eth`, `mordor`, `testnet-internal`
 ./bin/mantis cli generate-key-pairs 5
 ```
 
+- encrypt private key (could be used to setup private faucet)
```

Comment:
Probably bit more documentation (and explanation) would be helpful. Either here or in the help for that command